packages/ng/inline-message/inline-message.component.html

```diff
@@ -1 +1,11 @@
-<lu-icon [icon]="state" *ngIf="state && state !== 'default'" [alt]="state"></lu-icon>{{label}}
+@if (isStringPortalContent(label)) {
+<p class="inlineMessage-content">
+	<span *ngIf="state && state !== 'default'" class="lucca-icon inlineMessage-content-statusIcon" aria-hidden="true"></span>
+	{{label}}
+</p>
+} @else {
+<ng-container>
+	<span *ngIf="state && state !== 'default'" class="lucca-icon inlineMessage-content-statusIcon" aria-hidden="true"></span>
```

Comment:
I wonder if adding this `inlineMessage-content` is really necessary.
Also, it's unclear for me why in the second case `.inlineMessage-content-statusIcon` is outside `.inlineMessage-content`.

Reply:
To add a paragraph, there's no aria role possible: you absolutely have to use the tag. So, since the parent is a `lu-inline-message`, it's necessary to add the `p` as a child.